### ISSUE TITLE
wrong sign in both freqz() and phasez() ??

### ISSUE THREAD

**Issue by githubtomtom** [JuliaDSP/DSP.jl#380]
inside `response.jl`:
`ejw = exp(-im * w)` found in `freqz()`, why take the negative sign?
also,
`-atan.(imag(h), real(h)` found in `phasez()`, is it because of the (wrong) negative sign above ???

thanks.

**Comment by githubtomtom** [comment]
now I found that to get the correct phase response, one should:
```
function phasez_corrected(obj::DSP.Filters.PolynomialRatio, ω::AbV)
    ### issues in freqz() and phasez(): need to reverse coefficients to get correct (negated) phase response
    objrev = DSP.PolynomialRatio(DSP.coefb(obj)[end:-1:1], DSP.coefa(obj)[end:-1:1])
    phase = -angle.(DSP.freqz(objrev, ω))
    return phase
end
```

**Comment by galenlynch** [comment]
This is a known problem (#118, #135, #310) stemming from DSP.jl attempting to represent both continuous time and discrete time filters using the same `PolynomialRatio` type. I believe that your proposed change would break other things (see #310). @martinholters has a PR (#284) that should fix these and other problems, but it hasn't yet been reviewed or merged.

**Comment by githubtomtom** [comment]
Yes, seems like the problems have been around for years...
The above code is just a quick fix for someone who needs to use a correct `phasez()` now...

**Comment by martinholters** [comment]
Fixed by #284.
